tests/behat/features/gridfield-search.feature

```diff
@@ -4,21 +4,53 @@ Feature: Search in GridField
   So that I see proper result and don't see warning
 
   Background:
-    Given the "Company" "Walmart"
-    And the "Company" "ExxonMobil"
-    And the "Company" "Test"
+    Given the "Company" "Walmart" with "Category"="Retail"
+    And the "Employee" "Alen" with "Company"="1"
+    And the "Employee" "Bill" with "Company"="1"
+    And the "Employee" "Ford" with "Company"="1"
+    And the "Company" "ExxonMobil" with "Category"="Oil"
+    And the "Company" "Test" with "Category"="Other" 
     And I am logged in with "ADMIN" permissions
     And I go to "/admin/test"
-    
+ 
   Scenario: I can search and go to item
     When I press the "Open search and filter" button
     And I press the "Advanced" button
     Then I should see a "#Form_CompaniesSearchForm_Search_Name.no-change-track" element
     And I fill in "SearchBox__Name" with "Walmart"
     And I press the "Enter" key in the "SearchBox__Name" field
     Then I should see "Walmart" in the "#Form_EditForm" element
-    But I should not see "ExxonMobil" in the "#Form_EditForm" element
-    And I should not see "Test" in the "#Form_EditForm" element
+    But I should not see "ExxonMobil" in the ".col-Name" element
+    And I should not see "Test" in the ".col-Name" element
```

Comment:
There are multiple `.col-Name` elements and this probably only checks the first one. Check for it anywhere on the page instead.
"Test" is a hard one since it exists in the side admin area and breadcrumbs - maybe pick something more unique? I've suggested "Test Company" below (you'll need to change that elsewhere in this feature as well)
```suggestion
    But I should not see "ExxonMobil"
    And I should not see "Test Company"
```

Reply:
Please also do the same for the other `.col-Name` references below.

Reply:
I run behat tests in my local and it didn't work. Can do tests again.

Reply:
I returned `And I should not see "Test" in the ".col-Name" element `, since `And I should not see "Test Company"` fails in GitHub actions.